src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.spec.ts

```diff
@@ -66,6 +67,7 @@ describe('DotEditLayoutComponent', () => {
                 PaginatorService,
                 SocketFactory,
                 DotEditLayoutService,
+                DotEventsService,
```

Comment:
you must add a new test here to test the DptEventService used

Reply:
This is not used in this component at all, just added because the a child use it, actually I think the provider of this service need to go in the DOTTestBed since is global.